packages/settings/src/screens/license/index.tsx

```
@@ -115,18 +110,16 @@ const License = () => {
 				<div className="wcpos:sm:text-right wcpos:text-8xl">🎉</div>
 				<div className="wcpos:col-span-2">
-					<h3>{t('Thank You!', { _tags: 'wp-admin-settings' })}</h3>
+					<h3>{t('Thank You!')}</h3>
 					<p>
-						<T
-							_str="License {number} has been activated."
-							_tags="wp-admin-settings"
-							number={<code>{truncate(isString(data?.key) ? data?.key : '')}</code>}
+						<Trans
+							i18nKey="License <1>{number}</1> has been activated."
+							components={[, <code />]}
```

Biome (2.3.13)

[error] This array contains an empty slots..

The presences of empty slots may cause incorrect information and might be a typo.
Unsafe fix: Replace hole with undefined


(lint/suspicious/noSparseArray)